manifests/modelserving/servingruntimes-template.yaml

```diff
@@ -0,0 +1,56 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: ovms
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+    opendatahub.io/configurable: 'true'
+  annotations:
+    tags: 'ovms,servingruntime'
+    description: 'OpenVino Model Serving Definition'
+objects:
+  - apiVersion: serving.kserve.io/v1alpha1
+    kind: ServingRuntime
+    metadata:
+      annotations: []
+      labels:
+        opendatahub.io/dashboard: 'true'
+    spec:
+      builtInAdapter:
+        memBufferBytes: 134217728
+        modelLoadingTimeoutMillis: 90000
+        runtimeManagementPort: 8888
+        serverType: ovms
+      containers:
+        - args:
+            - '--port=8001'
+            - '--rest_port=8888'
+            - '--config_path=/models/model_config_list.json'
+            - '--file_system_poll_wait_seconds=0'
+            - '--grpc_bind_address=127.0.0.1'
+            - '--rest_bind_address=127.0.0.1'
+          image: >-
+            quay.io/modh/openvino-model-server@sha256:c89f76386bc8b59f0748cf173868e5beef21ac7d2f78dada69089c4d37c44116
```

Comment:
the openvino_model_server location and hash is different in odh-manifests model-mesh [params.env](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/base/params.env) than it is here. Is that intended?

Reply:
Also, instead of directly referencing the image here, it would be better to either reference it in [base kustomization.yaml](https://github.com/opendatahub-io/odh-dashboard/pull/1210/files#diff-73c2075d2639a2fcae7891d17241ef66a34bd7f65380a494db461497b9d69399) in the style of [this ](https://github.com/opendatahub-io/odh-manifests/blob/master/odh-dashboard/base/kustomization.yaml#L25). i.e. change this file to image: openvino-model-server and then in base/kustomization.yaml, reference it.

Reply:
I'm adding the same image as the one located in [model-serving](https://github.com/opendatahub-io/odh-manifests/commit/59e7748d1a588df4bcfba2982d401c80d48c34cd#diff-acee3f7c40c8bbcaedefbb5d66447f7828c7cc5b5b97e9fc4513f19ec394177eR6) I think we had some miscommunications about this topic.
@Xaenalt @taneem-ibrahim Can you verify if this image (and the spec) is the correct one to deliver as OOTB?

Reply:
out of the box default / OOTB, yeah